Modules/Sources/WordPressUI/Extensions/UIView+AutoLayout.swift

```diff
@@ -0,0 +1,100 @@
+import UIKit
+import SwiftUI
+
+extension UIView {
+    /// Pins edges of the view to the edges of the given container. By default,
+    /// pins to the superview.
+    @discardableResult
+    public func pinEdges(
+        _ edges: Edge.Set = .all,
+        to container: AutoLayoutItem? = nil,
```

Comment:
The "container" isn't the best term to use because you can pin the edges to a layout guide `view.pinEdges(to: view.safeAreaLayoutGuide)` or to a sibling view (not a container).